boa/src/object/mod.rs

```diff
@@ -863,6 +875,18 @@ impl Object {
         }
     }
 
+    /// Checks if it a `TypedArray` object.
+    #[inline]
+    pub fn is_typed_array(&self) -> bool {
+        matches!(
+            self.data,
+            ObjectData {
+                kind: ObjectKind::RegExp(_),
```

Comment:
```suggestion
                kind: ObjectKind::TypedArray,
```
Just a copy-paste error